### ISSUE TITLE
Videos not displaying properly in allure

### ISSUE THREAD

**Issue by pmwebster** [presidenten/wdio-video-reporter#12]
**Describe the bug**
Videos are not being displayed in allure. I get a black box that doesn't play anything.
The video files are recorded / work.
When I curl the url for one of the video files, it is a link to the physical location of the text file on my hard drive.
 
**Log**
```
--- Patching allure report ---
Error during onRunnerEnd:The "path" argument must be one of type string, Buffer, or URL. Received type undefinedTypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be one of type string, Buffer, or URL. Received type undefined
    at Object.readdirSync (fs.js:781:3)
    at Video.onRunnerEnd (/Users/pmwebster/Sites/qa/helpwdio/node_modules/wdio-video-reporter/dist/wdio-video-reporter.js:346:10)
    at Video.WDIOReporter.on.runner (/Users/pmwebster/Sites/qa/helpwdio/node_modules/@wdio/reporter/build/index.js:181:12)
    at Video.emit (events.js:189:13)
    at Video.EventEmitter.emit (domain.js:441:20)
    at reporters.forEach.reporter (/Users/pmwebster/Sites/qa/helpwdio/node_modules/@wdio/runner/build/reporter.js:61:49)
    at Array.forEach (<anonymous>)
    at BaseReporter.emit (/Users/pmwebster/Sites/qa/helpwdio/node_modules/@wdio/runner/build/reporter.js:61:20)
    at Runner.run (/Users/pmwebster/Sites/qa/helpwdio/node_modules/@wdio/runner/build/index.js:188:19)
    at process._tickCallback (internal/process/next_tick.js:68:7)%
```

**To Reproduce**
Follow the quick start instructions. This is private product that isn't released to the public yet, so I can't post a public url.

**Expected behavior**
I click on 'Execution video' and a video loads.

**Environment (please complete the following information):**
 - **wdio-video-reporter version:** 1.4.1
 - **WebdriverIO version:** 5.9.4
 - **Mode:** standalone
 - **Node.js version:** 10.15
 - **NPM version:** 6.9
 - **Browser name and version:** Chrome 74
 - **Platform name and version:** OSX

**Desktop (please complete the following information):**
same as above

**Additional context**
I don't really see any errors in any of the logs for allure or webdriverIO. I also don't see any network / console errors in chrome's debug mode of allure. The videos get created and play just fine in Quicktime, it feels like something isn't linking right from the browser to the underlying server maybe?

**Comment by presidenten** [comment]
Judging by the error, there seems to be something wrong with the [allure output dir](https://github.com/presidenten/wdio-video-reporter/blob/6786202ddba036b0aaeceee50d4092a16d2a4cee/dist/wdio-video-reporter.js#L346)

Would you mind sharing your `wdio.conf.js` and `wdio-0-0-Video-reporter.log` after a run with `logLevel: 'debug'`?

I tried both with same node version, wdio 5.9.4 and 5.9.6 without being able to reproduce the issue with both the wdio-template and the docker boilerplate.

**Comment by presidenten** [comment]
Never mind! Found the error. I was using the incorrect default allure directory after all.
Ill make a fix shortly.

**Comment by presidenten** [comment]
Fixed in v1.4.3

**Comment by pmwebster** [comment]
Thanks so much @presidenten for fixing this so fast! 
It works beautifully now :D
🎉 🎉 🎉
